tests/page/page-request-fulfill.spec.ts

```diff
@@ -418,3 +419,45 @@ it('should override status when fulfilling from har', async ({ page, isAndroid,
   // 404 should fail the CSS and styles should not apply.
   await expect(page.locator('body')).toHaveCSS('background-color', 'rgba(0, 0, 0, 0)');
 });
+
+it('should fulfill with har response', async ({ page, isAndroid, asset }) => {
+  it.fixme(isAndroid);
```

Comment:
isElectron as well

Reply:
No need to I believe as it does work there.